### DISCUSSION TITLE
Please let me know about Block Reward and Blocks Mined

### DISCUSSION THREAD

**Discussion by hsjw12** [filecoin-project/lotus#4632] [Storage Provider] [answered]
Please help me. How can I get Block Reward and how can I do Blocks Mined?
Please let me know about this.

![제목 없음](https://user-images.githubusercontent.com/70571701/97512684-7a8d4200-19cd-11eb-8f00-97e8e87ff8dd.png)

**Comment by jennijuju** [comment, marked as answer]
25% of block reward is going to your miners right after you mined a block and the rest is vested to you linearly 180 days. you can withdraw any available balance from your miner using `lotus-miner actor withdraw`
